qa/tasks/ceph_test_case.py

```diff
@@ -13,7 +14,32 @@
 class TestTimeoutError(RuntimeError):
     pass
 
-class CephTestCase(unittest.TestCase):
+class RunCephCmd:
+
+    def run_ceph_cmd(self, *args, **kwargs):
```

Comment:
Backport PR for this code was created long ago #54945.

I had also created PR #54926, which is a smaller version of PR #54945. But you had asked me to keep the bigger version so that future QA backports are easier.